server/pbench/bin/pbench-unpack-tarballs

```diff
@@ -90,7 +90,7 @@ list=$tmp/$PROG.list
 # find command will not generate any result if the linksrc is empty, and because of that du will not get any result 
 # to calculate the size. In that scenario du generates the size of the current directory as output, to handle
 # that exclude the current directory.
-find $ARCHIVE/*/$linksrc -type l -name '*.tar.xz' -printf "%l\n" 2>/dev/null | grep -v DUPLICATE | xargs du -b --exclude "." 2>/dev/null | sort -n > $list
+find $ARCHIVE/hostanmes/*/$linksrc -type l -name '*.tar.xz' -printf "%l\n" 2>/dev/null | grep -v DUPLICATE | xargs du -b --exclude "." 2>/dev/null | sort -n > $list
```

Comment:
This should only be done in the new version: the old version should remain unchanged.